webapp/lib/controller.dart

```diff
@@ -173,18 +196,18 @@ enum TagOperation {
   REMOVE
 }
 
-void _addTag(String tag, model.TagStyle tagStyle, Map<String, model.TagStyle> tagCollection, [bool isEditable = false]) {
+void _addTag(String selectedPackage, String tag, model.TagStyle tagStyle, Map<String, model.TagStyle> tagCollection, [bool isEditable = false]) {
```

Comment:
`selectedPackage` is already stored as global state, and it should match the selected package being shown in the view. Can we not avoid passing it as an argument to these helper functions, or am I missing something?

Reply:
I agree. I think I forgot to remove them once I made `selectedPackage` global. Removed.